File: pkaur6.java
// Prabhnoor Kaur
// Lab #6

public class pkaur6 {

   public static void main(String[] args) {
   
     	// Declare variables
        int x = 20, y = 10;
        double dx = 20.0, dy = 10.0;

        // Evaluate expressions for integer values
        System.out.println("Integer Evaluations:");
        System.out.println("x + y * x / y - x = " + (x + y * x / y - x));
        System.out.println("-x - y / x * y + x = " + (-x - y / x * y + x));
        System.out.println("x + y - x / y = " + (x + y - x / y));

        // Evaluate expressions for double values
        System.out.println("\nDouble Evaluations:");
        System.out.println("dx + dy * dx / dy - dx = " + (dx + dy * dx / dy - dx));
        System.out.println("-dx - dy / dx * dy + dx = " + (-dx - dy / dx * dy + dx));
        System.out.println("dx + dy - dx / dy = " + (dx + dy - dx / dy));
    }
}

File: pkaur6.c
// Prabhnoor Kaur
// Lab #6

#include <stdio.h>

int main(void)	
{

    // Declare variables
    int x = 20, y = 10;
    double dx = 20.0, dy = 10.0;

    // Evaluate expressions for integer values
    printf("Integer Evaluations:\n");
    printf("x + y * x / y - x = %d\n", x + y * x / y - x);
    printf("-x - y / x * y + x = %d\n", -x - y / x * y + x);
    printf("x + y - x / y = %d\n", x + y - x / y);


    // Evaluate expressions for double values
    printf("\nDouble Evaluations:\n");
    printf("dx + dy * dx / dy - dx = %lf\n", dx + dy * dx / dy - dx);
    printf("-dx - dy / dx * dy + dx = %lf\n", -dx - dy / dx * dy + dx);
    printf("dx + dy - dx / dy = %lf\n", dx + dy - dx / dy);

    return 0;

}

